regression/pages/lms/lms_home_page.py

```diff
@@ -0,0 +1,18 @@
+"""
+LMS Home page
+"""
+from bok_choy.page_object import PageObject
+from regression.pages.lms import BASE_URL
+
+
+class LmsHome(PageObject):
```

Comment:
Can we extend it from https://github.com/edx/edx-platform/blob/master/common/test/acceptance/pages/lms/index.py ?